app/api/cruds/utils.py

```diff
@@ -18,3 +18,17 @@ def get_user(relayId, otherwise=None):
         return User.objects.get(pk=from_global_id(relayId)[1])
     except:
         return otherwise
+
+
+def get_object(object_name, relayId, otherwise=None):
+    try:
+        return object_name.objects.get(pk=from_global_id(relayId)[1])
+    except:
+        return otherwise
```

Comment:
Compared to `get_user`, this is more general that could be used for every model instance - I think it's a better solution in the sense that it's reusable. Though you would be doing more than required of the task, I think it would save us a great deal if you could get ride of `get_user` and update everywhere it was called with `get_object`.